agot-bg-game-server/src/common/ingame-game-state/action-game-state/resolve-march-order-game-state/combat-game-state/before-combat-house-card-abilities-game-state/aeron-damphair-dwd-ability-game-state/AeronDamphairDwDAbilityGameState.ts

```diff
@@ -8,11 +7,14 @@ import {ClientMessage} from "../../../../../../../messages/ClientMessage";
 import {ServerMessage} from "../../../../../../../messages/ServerMessage";
 import IngameGameState from "../../../../../IngameGameState";
 import BeforeCombatHouseCardAbilitiesGameState from "../BeforeCombatHouseCardAbilitiesGameState";
-
+import BiddingGameState, { SerializedBiddingGameState } from "../../../../../westeros-game-state/bidding-game-state/BiddingGameState";
 
 export default class AeronDamphairDwDAbilityGameState extends GameState<
-    BeforeCombatHouseCardAbilitiesGameState["childGameState"], SimpleChoiceGameState
-> {
+    BeforeCombatHouseCardAbilitiesGameState["childGameState"],
+    BiddingGameState<AeronDamphairDwDAbilityGameState>> {
```

Comment:
I would not extend AeronDamphairDwDAbilityGameState with the bidding game state. I would extend the AeronDamphairDwDAbilityComponent with a slider and do the UI handling there.